### ISSUE TITLE
Provide an option to select a device, when multiple devices are connected

### ISSUE THREAD

**Issue by Kikju** [Genymobile/scrcpy#1965]
Often I have connected multiple emulators and phones to my computer. I have to open separate terminal, list the devices via `adb devices` and then copy each serial number and run scrcpy with `-s` argument. Or often I just launch scrcpy without arguments, and I get an error `adb: error: failed to get feature set: more than one device/emulator`.
It would be helpful, to provide an option to choose a specific device and continue instead of error "more than one device/emulator".

It could looks like this:
```
Multiple devices detected. To continue, please type the order of device you want to launch:
1. emulator-5554
2. 0123456789abcdef
```

Then I could just type "2" to continue and open scrcpy with selected device.

 - [x] I have checked that a similar [feature request](https://github.com/Genymobile/scrcpy/issues?q=is%3Aopen+is%3Aissue+label%3A%22feature+request%22) does not already exist.

**Comment by PonyPC** [comment]
I agree with adding this feature, this also occurred while enabled wifi adb connection in one device.
But there are several applications that provide a user friendly UI such as QtScrcpy you can try: https://github.com/barry-ran/QtScrcpy
It is enough for developing android apk though some bugs in listing devices on my desktop.
I prefer this software AnLink https://anl.ink which seems use same technology from scrcpy, it is more convenient than QtScrcpy but not open source software, so choose your favorite.
Wish scrcpy could add an UI to choose devices.

**Comment by erawhctim** [comment]
Would love to see something like this implemented. Another option (that mimics `adb` behavior) would be to add `-d` and `-e` flags to indicate a device vs. emulator?

**Comment by rom1v** [comment]
Oh, I forgot this feature request.

I implemented such a feature this week-end: please review #3005.

**Comment by rom1v** [comment]
Fixed by #3005.
